src/avatar/Avatar.tsx

```diff
@@ -0,0 +1,35 @@
+import "./_avatar.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+export interface AvatarProps {
+  alt: string;
+  placeholderSrc: string;
+  size?: {
+    width: string;
+    height: string;
+  };
```

Comment:
We always have the avatar as a circle so we can just get a `size` 
```suggestion
  size?: number;
```